cactusbot/commands/magic/alias.py

```diff
@@ -31,3 +31,19 @@ class Alias(Command):
             return "Alias {} removed.".format(alias)
         elif response.status == 404:
             return "Alias {} doesn't exist!".format(alias)
+
+    @Command.command("list", role="moderator")
+    async def list_aliases(self):
+        """List all aliases."""
+        response = await self.api.get_command()
+
+        if response.status == 200:
+            commands = (await response.json())["data"]
```

Comment:
Is this a good idea? What if, for whatever reason (should never happen but could), `data` isn't a key in the response?